packages/idos-sdk-js/src/lib/idos.ts

```diff
@@ -63,7 +66,13 @@ export class idOS {
       new IframeEnclave(enclaveOptions ?? { container: container! }),
     );
 
-    this.data = new Data(kwilWrapper, this.enclave);
+    this.data = new Data(
+      kwilWrapper,
+      this.enclave,
+      new AttributeDataLayer(this.enclave, kwilWrapper),
+      new CredentialDataLayer(this.enclave, kwilWrapper),
+      new WalletDataLayer(this.enclave, kwilWrapper),
+    );
```

Comment:
Please make the order of `kwilWrapper` and `this.enclave` consistent between `Data` and `DataLayer` children. Just trying to avoid mistakes.